conda-forge.yml

```diff
@@ -1,5 +1,10 @@
-channel_priority: strict
-azure:
-  settings_win:
-    pool:
-      vmImage: windows-2019
\ No newline at end of file
+github:
+  branch_name: main
```

Comment:
Please don't touch files outside of `recipes/`

Reply:
I needed these changes for `roaring-landmask` to get it to build for arm64 and aarch64, but I can fix that later if it doesn't work.

Reply:
Removed these changes, think this should be ready for another pass. Thanks for your comments so far.